src/main/java/com/redhat/darcy/ui/Elements.java

```diff
@@ -22,18 +22,7 @@
 import com.redhat.darcy.ui.api.HasElementContext;
 import com.redhat.darcy.ui.api.Locator;
 import com.redhat.darcy.ui.api.WrapsElement;
-import com.redhat.darcy.ui.api.elements.Button;
-import com.redhat.darcy.ui.api.elements.Checkbox;
-import com.redhat.darcy.ui.api.elements.DateInput;
-import com.redhat.darcy.ui.api.elements.Element;
-import com.redhat.darcy.ui.api.elements.FileSelect;
-import com.redhat.darcy.ui.api.elements.Label;
-import com.redhat.darcy.ui.api.elements.Link;
-import com.redhat.darcy.ui.api.elements.MultiSelect;
-import com.redhat.darcy.ui.api.elements.Radio;
-import com.redhat.darcy.ui.api.elements.Select;
-import com.redhat.darcy.ui.api.elements.Text;
-import com.redhat.darcy.ui.api.elements.TextInput;
+import com.redhat.darcy.ui.api.elements.*;
```

Comment:
We've avoided * imports as a matter of style to be explicit; let's keep that rule I think. You may have to configure your IDE to avoid * imports.